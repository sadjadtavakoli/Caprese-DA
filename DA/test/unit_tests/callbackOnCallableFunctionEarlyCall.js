let fs = require('fs')

let callbackFunction = function (item) {
    // console.log(item)
}

let caller = function(){    
    let items = [1, 2]
    items.forEach(callbackFunction)    
}
callbackFunction("test")
caller()