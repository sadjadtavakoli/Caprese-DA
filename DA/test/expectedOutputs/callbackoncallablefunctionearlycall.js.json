{"callbackFunction-test/unit_tests/callbackoncallablefunctionearlycall.js-3-5":{"impacted":["test/unit_tests/callbackoncallablefunctionearlycall.js"]},"test/unit_tests/callbackoncallablefunctionearlycall.js":{"impacted":["callbackFunction-test/unit_tests/callbackoncallablefunctionearlycall.js-3-5"]},"caller-test/unit_tests/callbackoncallablefunctionearlycall.js-7-10":{"impacted":["test/unit_tests/callbackoncallablefunctionearlycall.js"]}}